src/LinkDotNet.Blog.Web/Features/Bookmarks/IBookmarkService.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using LinkDotNet.Blog.Domain;
+
+namespace LinkDotNet.Blog.Web.Features.Bookmarks;
+
+public interface IBookmarkService
+{
+    public Task<bool> IsBookMarked(string postId);
```

Comment:
I would also write `IsBookmarked` as you did not use capital `M` for the other methods.